core/trino-main/src/test/java/io/trino/sql/planner/TestConnectorExpressionTranslator.java

```diff
@@ -100,6 +100,19 @@ public void testTranslationToConnectorExpression()
                         List.of(new Variable("varchar_symbol_1", VARCHAR_TYPE),
                                 new Constant(Slices.wrappedBuffer(pattern.getBytes(UTF_8)), createVarcharType(pattern.length()))))));
 
+        String escape = "\\";
+        assertTranslationToConnectorExpression(
+                TEST_SESSION,
+                new LikePredicate(
+                        new SymbolReference("varchar_symbol_1"),
+                        new StringLiteral(pattern),
+                        Optional.of(new StringLiteral(escape))),
+                Optional.of(new Call(BOOLEAN,
+                        LIKE_PATTERN_FUNCTION_NAME,
+                        List.of(new Variable("varchar_symbol_1", VARCHAR_TYPE),
```

Comment:
put each arg on own line (add line break before `new Variable`)

(yes, the code above doesn't conform to the code style)